### PR TITLE
129 - Creation of dynamic property MatomoTracker::$currentTs is deprecated

### PR DESCRIPTION
**Pull request by lutdev**
### Description:

I created properties to fix the problem with the dynamic properties. All default values I took from the `__construct()`
Issue: https://github.com/matomo-org/matomo-php-tracker/issues/129

I would be happy to update CHANGELOG but have no idea how my updates should look like.

### CODE REVIEW

**Comment by michalkleiner** [comment]
Thanks for starting the PR @lutdev. Is there a reason to have the properties public? Are they used outside of the class? I'd imagine they can be private, protected at most. What do you think?

**Comment by lutdev** [comment]
@michalkleiner you are welcome :) The main reason to make not breaking changes. If the properties will be public, developers still can use them as they use them now. Later we can try to make them private/protected step by step.

File: MatomoTracker.php [hunk 1]
```diff
@@ -135,43 +227,12 @@ public function __construct($idSite, $apiUrl = '')
...
-        // Allow debug while blocking the request
-        $this->requestTimeout = 600;
-        $this->requestConnectTimeout = 300;
-        $this->doBulkRequests = false;
-        $this->storedTrackingActions = [];
-
-        $this->sendImageResponse = true;
-
+        
         $this->visitorCustomVar = $this->getCustomVariablesFromCookie();
```
Comment (sgiehl) [inline MatomoTracker.php:235]: Doesn't this one need a property declaration as well?

Comment (lutdev) [reply MatomoTracker.php:237]: @sgiehl yes, I missed it. Thank you!)

**Pull request merged**